tests/jenkins/pages/treeherder.py

```diff
@@ -160,27 +99,6 @@ def filter_by(self, term, method='pointer'):
         else:
             raise Exception('Unsupported method: {}'.format(method))
 
-    def filter_job_superseded(self):
-        self.find_element(*self._nav_filter_superseded_locator).click()
-
-    def filter_job_failures(self):
-        self.find_element(*self._nav_filter_failures_locator).click()
-
-    def filter_job_in_progress(self):
-        self.find_element(*self._nav_filter_inprogress_locator).click()
-
-    def filter_job_retries(self):
-        self.find_element(*self._nav_filter_retry_locator).click()
-
-    def filter_job_successes(self):
-        self.find_element(*self._nav_filter_successes_locator).click()
-
-    def filter_job_usercancel(self):
-        self.find_element(*self._nav_filter_usercancel_locator).click()
-
-    def filter_unclassified_jobs(self):
-        self.find_element(*self._unclassified_failure_filter_locator).click()
```

Comment:
I think `_unclassified_failure_filter_locator` is unused now?

Reply:
It's still used by `test_view_unclassified_jobs` for now, but it will be migrated soon.

https://github.com/mozilla/treeherder/blob/578e186fe966c118b430f970fe5a7b497cfdfda9/tests/jenkins/tests/test_unclassified_job.py#L17-L28

Reply:
Oh, wait... I shouldn't have removed this method! 😆